flink-python/pyflink/datastream/tests/test_data_stream.py

```diff
@@ -1068,6 +1013,35 @@ def on_timer(self, timestamp, ctx):
         expected_result.sort()
         self.assertEqual(expected_result, result)
 
+    def test_reduce_function_without_data_types(self):
```

Comment:
Could this add `test_reduce_function_with_data_types` test case?

Reply:
I'll rename it to test_reduce_function to avoid confusion.